homeassistant/components/v2c/const.py

```diff
@@ -1,3 +1,5 @@
 """Constants for the V2C integration."""
 
 DOMAIN = "v2c"
+MIN_INTENSITY = 6
+MAX_INTENSITY = 32
```

Comment:
These are only used once so they don't really belong in const.py

Reply:
There are 2 more numbers that will be later added, should they still be defined in number.py ?

Reply:
I mean, const is a place for shared constants, in this case you're only using the constant in 1 file, so it doesn't really belong in const.py